exercise_01.py

```diff
@@ -0,0 +1,18 @@
+def displayPattern(n):
```

Comment:
Now that we are following PEP-8, please use the convention `lower_case_with_underscores` to name variables and functions instead of `lowerCamelCase`.